deps/wazuh_testing/wazuh_testing/end_to_end/vulnerability_detector.py

```diff
@@ -131,6 +131,34 @@ def get_alerts_by_agent(alerts, regex) -> Dict:
     return alerts_vuln_by_agent
 
 
+def parse_vulnerability_detector_alerts(alerts) -> Dict:
+    """
+    Get specific alerts by agent.
+
+    Args:
+        alerts (list): List of alerts.
+        regex (str): Regular expression to match the alerts.
```

Comment:
`regex` is not an argument

Reply:
Done 9c2fb39d9da5b0dd0a337706edaaebff0e49e369